manual/de/ch_Tools_Assistants.xml

```diff
@@ -5324,15 +5323,57 @@
           </glossentry>
         </glosslist>
       </para>
+
+      <para>Der Vollständigkeit halber sind hier noch einige weitere Kursarten genannt, welche zwar
+        durch &app-fq; ermittelt, jedoch in &app; nicht weiter verwendet werden.
```

Comment:
– dienen in erster Linie der  [Chart-Analyse](https://de.wikipedia.org/wiki/Technische_Analyse) und sprengen damit den Rahmen von GnuCash.